tests/src/test/scala/system/basic/WskConsoleTests.scala

```diff
@@ -88,7 +90,10 @@ abstract class WskConsoleTests extends TestHelpers with WskTestHelpers {
   it should "show repeated activations" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
     val name = withTimestamp("countdown")
     assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-      action.create(name, Some(TestUtils.getTestActionFilename("countdown.js")))
+      action.create(
+        name,
+        Some(TestUtils.getTestActionFilename("countdown.js")),
+        annotations = Map(WhiskAction.provideApiKeyAnnotationName -> JsBoolean(true)))
```

Comment:
i think this test suite should move entirely to the CLI IMO. 
that said, why does the test not fail here already 😲 ?

i agree the annotation should be present for countdown to work since it does an action invoke.

Reply:
@rabbah, the test suite is an abstract class. I don't believe this suite gets ran outside of the CLI repo. Which explains why it did not fail anywhere else.